docs/src/content/docs/es/guides/sidebar.mdx

```diff
@@ -310,8 +363,7 @@ starlight({
 			items: [
 				// Un enlace con una insignia "Stub" amarilla.
 				{
-					label: 'Sirio',
-					link: '/stars/sirius/',
+					slug: 'stars/sirius',
```

Comment:
The highlight of this code block should be updated from `10` to `9` to match this change.